source/hardware/connecting-to-the-serial-console.rst

```diff
@@ -93,6 +93,10 @@ port, start PuTTY and then set the following parameters:
 The session may be saved for later use. Click **Open** to start the
 serial console session.
 
+MINICOM
+~~~~~
```

Comment:
Header characters must be the same length as the header. It's two characters short. Also needs a blank line after.

Reply:
Ok thanks you for the comment, i've fix that :)